draft repo setup (#47)
carpentries-incubator/jekyll-pages-novice: _episodes/01-introduction.md

> {: .solution }
{: .challenge }

> ## Setting up a repository

Author: ```suggestion
## Setting Up a Repository
```

a version control software called `git` to track changes to the files.
For our purposes we will mostly be ignoring the version control software,
though it can be handy if you need to revert to old versions.
In this lesson we will be working with this folder on the web to control
the website we will be creating.

> ## Setup a GitHub account
> Before you can create a repo, you will need to
> [create a GitHub account](https://github.com/join)
{: .callout}

Click the green "New" repo button on the left hand side of github

![Green New Repo Button](../fig/new_repo_button.png)

**or** click the "+" menu in the upper righthand corner and
choose "New Repository".

![Dropdown plus menu with new repository option highlighted](../fig/plus_new_repo.png)

Next you will need to fill in some info about your repository.

![Blank new repository page](../fig/blank_new_repo.png)

In this lesson, we will be working on a general group website.
You can imagine this website may be for your lab group, a specific project group,
or another group you work with.
Under the "Repository name" field type `group-website`.

![Repository name set to group-website](../fig/set_repo_name.png)

> So we know what this repo is when we find it later we can also add a description.

Reviewer: ```suggestion
So we know what this repo is when we find it later, we can also add a description.
```

Reviewer 2: For me as non-native speaker, this sentence is hard to understand.
Does the following have the same meaning?
"To tell our future-self what this repo is about, we can also add a description."
"To know what this repo is about when we find it later, we can also add a description. "

Author: Maybe we should flip the order of the phrases.  Perhaps...
```suggestion
We can also add a description so we know what this repo is when we find it again after the workshop.
```
would be better?

Reviewer 3: I like this but wonder if it would be even clearer if it read "[...] when we find it **again** after the workshop."
